### ISSUE TITLE
Add equihash solver 48,5

### ISSUE THREAD

**Issue by marcofortina** [mbevand/silentarmy#108]
Hi there

is it possible to add the the equihash solver 48,5 for regtest solutions generation?

Thanks,
Marco

**Comment by roycewilliams** [comment]
The maintainer (not me) is not currently maintaining silentarmy.

**Comment by marcofortina** [comment]
uhm? Is this project abandoned?

**Comment by roycewilliams** [comment]
I'm not positive. Abandoned is a strong word, though - maybe "on long-term hold" is closer.

**Comment by mbevand** [comment]
Correct, I no longer maintain SILENTARMY. I just added a note in the readme about it.

**Comment by mocmoctoddotcomcom** [comment]
I've noticed this with a lot of miners.  After compiling and testing just about every miner available, it seems that once the bugs get worked out, somehow something no longer works, then somebody releases a fork with a "dev fee" kickback and everybody just uses that but without the ability to edit anything without reverse engineering it.

**Comment by miranos** [comment]
Does anyone solved this issue yet ?
